app/src/main/java/org/simple/clinic/teleconsultlog/teleconsultrecord/TeleconsultRecord.kt

```diff
@@ -27,9 +29,53 @@ data class TeleconsultRecord(
     val teleconsultRecordInfo: TeleconsultRecordInfo?,
 
     @Embedded
-    val timestamp: Timestamps
+    val timestamp: Timestamps,
+
+    val syncStatus: SyncStatus
 ) {
 
+  fun toPayload(): TeleconsultRecordPayload {
+    val teleconsultRequestInfoPayload = getTeleconsultRequestInfoPayload()
+    val teleconsultRecordInfoPayload = getTeleconsultRecordInfoPayload()
+
+    return TeleconsultRecordPayload(
+        id = id,
+        patientId = patientId,
+        medicalOfficerId = medicalOfficerId,
+        teleconsultRequestInfo = teleconsultRequestInfoPayload,
+        teleconsultRecordInfo = teleconsultRecordInfoPayload,
+        createdAt = timestamp.createdAt,
+        updatedAt = timestamp.updatedAt,
+        deletedAt = timestamp.deletedAt
+    )
+  }
+
+  private fun getTeleconsultRecordInfoPayload(): TeleconsultRecordInfoPayload? {
+    return if (teleconsultRecordInfo != null) {
+      TeleconsultRecordInfoPayload(
+          recordedAt = teleconsultRecordInfo.recordedAt,
+          teleconsultationType = teleconsultRecordInfo.teleconsultationType,
+          patientTookMedicines = teleconsultRecordInfo.patientTookMedicines,
+          patientConsented = teleconsultRecordInfo.patientConsented,
+          medicalOfficerNumber = teleconsultRecordInfo.medicalOfficerNumber.toString(),
```

Comment:
I just realised that this `.toString()` call on a nullable string will return the string `"null"` if it is `null`, which is incorrect behaviour and we should be sending a blank string instead. The `.orEmpty()` extension method is the right one to use here.